client/components/homepage/Contact.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+const Contact = () => (
+  <section id="contact" className="section background-gray-lightest">
+    <div className="container">
+      <div className="row">
+        <div className="col-md-3">
+          <div className="box-simple">
+            <div className="icon" />
+            <h3>Name</h3>
+            <p>Profile Info Summary</p>
+            <p>Github Link</p>
+            <p>LinkedIn Link</p>
+          </div>
+        </div>
+        <div className="col-md-3">
+          <div className="box-simple">
+            <div className="icon" />
+            <h3>Name</h3>
+            <p>Profile Info Summary</p>
+            <p>Github Link</p>
+            <p>LinkedIn Link</p>
+          </div>
+        </div>
+        <div className="col-md-3">
+          <div className="box-simple">
+            <div className="icon" />
+            <h3>Name</h3>
+            <p>Profile Info Summary</p>
+            <p>Github Link</p>
+            <p>LinkedIn Link</p>
+          </div>
+        </div>
+        <div className="col-md-3">
+          <div className="box-simple">
+            <div className="icon" />
+            <h3>Name</h3>
+            <p>Profile Info Summary</p>
+            <p>Github Link</p>
+            <p>LinkedIn Link</p>
```

Comment:
All of these contact sections seem similar enough that we should probably turn them into a component we can reuse and keep ourselves dry ;)